projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-product-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,81 @@
+import * as configuration from '../../helpers/product-configuration';
+import * as configurationOverview from '../../helpers/product-configuration-overview';
+import { tabbingOrderConfig as tabConfig } from '../../helpers/accessibility/tabbing-order.config';
+import { verifyTabbingOrder } from '../../helpers/accessibility/tabbing-order';
+
+const testProduct = 'CONF_CAMERA_SL';
+
+const configurator = 'CPQCONFIGURATOR';
+
+const containerSelectorConfigForm = '.VariantConfigurationTemplate';
+const containerSelectorOverviewForm = '[class*=VariantConfig]';
+
+function goToConfigPage(configuratorType, product) {
+  cy.visit(
+    `/electronics-spa/en/USD/configure${configuratorType}/product/entityKey/${product}`
+  ).then(() => {
+    configuration.isConfigPageDisplayed();
+  });
+}
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  describe('Product Config Tabbing', () => {
+    it('should allow to navigate with tab key', () => {
+      goToConfigPage(configurator, testProduct);
+      configuration.isGroupMenuDisplayed();
+      configuration.isConfigHeaderDisplayed();
+
+      verifyTabbingOrder(
+        containerSelectorConfigForm,
+        tabConfig.productConfigurationPage
+      );
+
+      configuration.selectAttribute('CAMERA_MODE', 'radioGroup', 'P');
+
+      configuration.clickAddToCartBtn();
+      configurationOverview.isConfigOverviewPageDisplayed();
+
+      cy.get('cx-global-message').should('not.be.visible');
+      cy.get('cx-config-update-message').should('not.be.visible');
+      cy.get('.cx-config-add-to-cart-btn').should('be.visible');
+      cy.get('.cx-config-group-attribute').should('be.visible');
+
+      verifyTabbingOrder(
+        containerSelectorOverviewForm,
+        tabConfig.productConfigurationOverview
+      );
+    });
+  });
+
+  describe('Product Config Keep Focus', () => {
+    it('should keep focus after selection', () => {
+      goToConfigPage(configurator, testProduct);
+
+      configuration.selectAttribute('CAMERA_COLOR', 'radioGroup', 'METALLIC');
+
+      cy.wait(3000);
+
+      cy.focused().should(
+        'have.attr',
+        'id',
+        'cx-config--radioGroup--CAMERA_COLOR--METALLIC'
+      );
+
+      configuration.clickOnNextBtn('Specification');
+
+      configuration.selectAttribute('CAMERA_SD_CARD', 'checkBoxList', 'SDXC');
+
+      cy.wait(3000);
+
+      cy.focused().should(
```

Comment:
Replace through the newly defined method that proves whether the focus set as expected

Reply:
done